rudder-agent/SOURCES/debug-script/tests/rudder_runtime

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+#
+# runtime test
+#
+
+# Describe your tests here
+runtime_test_description="Verify if some runs took more than amount of time between runs"
+  
+runtime_test() {
+
+  # By default, this test is successful
+  RUNTIME_AGENT='ok'
+  LOG_PATH="/var/rudder/cfengine-community/outputs"
+  runtime=300
+  
+  if [ -e $LOG_PATH ]
+  then
+    for i in $(ls -rt ${LOG_PATH} | tail -5)
+    do  
+      # Get a last date access 
+      start_exec=$(echo $i | cut -d'_' -f6)
+      # Get a last date of mofification
+      date_modif=$(stat -c %Y ${LOG_PATH}/$i)
+      # fet a agent runtime
+      runtime_agent=$(($date_modif-$start_exec))
```

Comment:
You calculat 5 agent run times but only the last ont is kept in this variable, so there is only one test.